packages/devextreme/scss/widgets/fluent/loadIndicator/_index.scss

```diff
@@ -69,29 +69,34 @@
   }
 }
 
-.dx-loadindicator-segment0 {
-  right: 0;
+.dx-loadindicator-segment2 {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  overflow: hidden;
+  box-sizing: border-box;
 
   .dx-loadindicator-segment-inner {
-    left: -100%;
-    border-left-color: transparent;
-    transform: rotate(129deg);
-    animation: dx-right-segment-rotation 1333ms cubic-bezier(0.4, 0, 0.2, 1) infinite both;
+    border-color: #D0E5FB;
```

Comment:
We shouldn't to use hard-coded value of color in our styles. Please, use base variables for this